dask_jobqueue/core.py

```diff
@@ -407,5 +409,10 @@ def _del_pending_jobs(self):
         return jobs
 
     def _job_id_from_submit_output(self, out):
-        raise NotImplementedError('_job_id_from_submit_output must be implemented when JobQueueCluster is '
-                                  'inherited. It should convert the stdout from submit_command to the job id')
+        match = re.search(self.job_id_regexp, out)
+        job_id = match.group('job_id')
+        if job_id is None:
+            raise ValueError('Could not parse job id from submission command'
```

Comment:
Add a space at the end of each line otherwise the text will be something like "submission commandoutput"

Reply:
Done.